Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'rails', '~> 4.2'
 
 
 gem 'pg'
+gem 'sqlite3'
```

Comment:
This should be in the `:development, :test` section, so it doesn't clutter up production with a vestigal gem.